Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/ExportTests.cs

```diff
@@ -0,0 +1,50 @@
+using System.Threading.Tasks;
+using Xunit;
+using System.Linq;
+using VirtoCommerce.ExportModule.Core.Model;
+using VirtoCommerce.PricingModule.Core.Model;
```

Comment:
Cannot use Pricing module here. Probably we could remove this test from Export module (add it to Pricing module tests instead).